src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js

```diff
@@ -1,32 +1,47 @@
-import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
+import React from 'react';
 
 import HospitalizationPeriodView from '../components/HospitalizationPeriodView';
 
-import fullSchema from 'vets-json-schema/dist/21-526EZ-ALLCLAIMS-schema.json';
-import { unemployabilityTitle } from '../content/unemployabilityFormIntro';
+// import fullSchema from 'vets-json-schema/dist/21-526EZ-ALLCLAIMS-schema.json';
+import { unemployabilityTitleHospitalization } from '../content/unemployabilityFormIntro';
```

Comment:
Use `import { unemployabilityTitle } from '../content/unemployabilityFormIntro';` for the main title, See https://github.com/department-of-veterans-affairs/vets-website/blob/bah-8940-doctor-interview/src/applications/disability-benefits/all-claims/pages/unemployabilityCertification.js

Reply:
changed!